.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: ruby
+
+services:
+  - docker
+
+install: git clone --depth 1 https://github.com/sstephenson/bats.git
```

Comment:
Should `go-makefile` submodule `bats`? Or is the assumption that most people interested in running tests will already have `bats` installed?